modules/experiment/src/main/resources/schemas/neurosciencegraph/experiment/brainslicing/v0.1.1.json

```diff
@@ -0,0 +1,101 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/experiment/brainslicing/v0.1.0/shapes/"
+    }
+  ],
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.2",
```

Comment:
Use the [neurosciencegraph/commons/activity/v0.1.3](https://github.com/INCF/neuroshapes/blob/master/modules/commons/src/main/resources/schemas/neurosciencegraph/commons/activity/v0.1.3.json) schema instead